lib/android/src/main/java/com/airbnb/android/react/maps/AirMapModule.java

```diff
@@ -212,6 +212,9 @@ public void execute(NativeViewHierarchyManager nvhm)
         try {
           List<Address> list =
                   geocoder.getFromLocation(coordinate.getDouble("latitude"),coordinate.getDouble("longitude"),1);
+          if (list.isEmpty()) {
+            promise.reject("Can not get address location");
```

Comment:
Should we keep the error message the same across both platforms?

Reply:
Also, it might be wise to check for `longitude & `latitude` properties like we do on iOS alongside the check for an empty list.

Reply:
I believe that check is already happening earlier on, and won't get to this point unless valid coordinate data has been provided.

Reply:
@owinter86 I agree, can you add handler if empty response of `geocoder` https://github.com/owinter86/react-native-maps/blob/fix/geocoder/lib/ios/AirMaps/AIRMapManager.m#L602 with similar message. BTW, this is good to have. 👍

Reply:
A yea I see the logic here: https://github.com/react-native-maps/react-native-maps/blob/a0e3b393c3d5b4fb3bac03c7a4e4b427889624b3/lib/android/src/main/java/com/airbnb/android/react/maps/AirMapModule.java#L205-L208

@mechazod It looks like the conditional you had set up on Android is what @owinter86 is proposing on iOS. Maybe it was just an oversight?

Reply:
Nice @christopherdro good catch.